src/main/java/org/dcsa/bkg/repositories/BookingConfirmationRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.dcsa.bkg.repositories;
+
+import org.dcsa.core.events.model.Shipment;
+import org.dcsa.core.repository.ExtendedRepository;
+import reactor.core.publisher.Mono;
+
+public interface BookingConfirmationRepository

```

Comment:
is this file still needed? Seems like it is not used and instead the ShippingRepository from Event Core is used.

Reply:
Ah, you're right. I've deleted it.